javascript-data-model/order-history.js

```diff
@@ -0,0 +1,68 @@
+var orderHistory = [
+  {
+    number: '114-3941689-8772232',
+    total: '$34.00',
+    numberItems: 1,
+    orderDate: 'August 4, 2020',
+    deliveryDate: 'August 8, 2020',
+    recipient: 'JS Masher',
+    itemDescription: [
+      {
+        item1: 'book, $31.55, Javascript for impatient programmers'
```

Comment:
What is this? A string pretending to be an array?

Reply:
What is each part for? How is a developer supposed to use this data? This looks like three separate pieces of data glued together into one string with no explanation of what those pieces are. What data type in JavaScript is useful for storing multiple pieces of data about one thing?